daemon/src/maker_inc_connections.rs

```diff
@@ -417,26 +418,40 @@ impl Actor {
             .await;
 
         let mut tasks = Tasks::default();
-        tasks.add({
-            let this = this.clone();
-
-            async move {
-                while let Ok(Some(msg)) = read.try_next().await {
-                    let res = this
-                        .send(maker_cfd::FromTaker {
+        tasks.add_fallible(
```

Comment:
❓ Is the advantage of using `add_fallible` here that we are more explicit about the error case? Or is there a functional difference?

Reply:
It allows me to use `?` inside the future and handle it all in one spot!

Reply:
Ah, of course! Well worth it.